### PR TITLE
#140 | Ziyang | UI improvements

### LINKED ISSUE #140: UI improvements
**As a** Service User
**I need** a interactive and fluent UI
**So that** I can better understand the service

**Details & Assumptions:** 

* Search results animation and loading effect

**Acceptance Criteria:**

```gherkin
When I click the search button
Then I should see the animation effect
```

### PR DESCRIPTION
**Pull request by dizys**
Closes #140.

### CODE REVIEW

File: service/static/index.html [hunk 1]
```diff
@@ -371,32 +371,61 @@
...
-                    <div>
-                      <b>Status:</b> {{promotion.active ? 'active' :
-                      'inactive'}}
-                    </div>
-                    <div>
-                      <b>Product ID:</b> {{promotion.product_id || 'null'}}
+          <transition name="section">
+            <div class="promotions-loading" v-if="promotionsLoading">
+              <div class="lds-default promotions-loading-icon">
+                <div></div>
```
Comment (rladuddlf) [inline service/static/index.html:377]: Are these empty _div/div_ needed? line 377 to 388

**Review by rladuddlf — APPROVED** [review]
lgtm

File: service/static/index.html (hunk 1 above)
Comment (dizys) [reply service/static/index.html:377]: These are css loading tricks

Comment (dizys) [reply service/static/index.html:377]: For display css animations, they are circles so don't have text in them.

**Pull request merged**